ansible_collections/arista/avd/roles/eos_designs/docs/tables/ptp.md

```diff
@@ -25,6 +25,10 @@
 === "YAML"
 
     ```yaml
+    # The `ptp` setting is deprecated and it will be removed in version 5.0.0 to avoid conflict with `ptp` setting in eos_cli_config_gen. `ptp_settings` takes precedence when both `ptp` and `ptp_settings` are set.
```

Comment:
should we move ptp_profiles documentation also in ptp_settings.md
@ClausHolbechArista 

[line number 37](https://github.com/aristanetworks/avd/pull/4155/files#diff-be639d3b605ad40103adc39c4cccb48fc9d17a81e66a4b9bc4b84c6067399b00L33)

Reply:
Moved.